src/ImageGallery.jsx

```diff
@@ -856,19 +858,32 @@ export default class ImageGallery extends React.Component {
     };
   }
 
+  _getThumbnailContainerClassName() {
+    let className;
+    const { isRTL, thumbnailPosition } = this.props;
+
+    if (this._isThumbnailHorizontal() && isRTL) {
+      return thumbnailPosition==='left' ? 'right' : 'left';
```

Comment:
Is this intentional? If I set the thumbnail bar to the `left` in a `RTL` environment  is it suppose to show up on the `right`?

Reply:
Hello, sorry for being late I was busy a little bit.
No, it's not intentional, this is my bad.
In `RTL` environment the direction of thumbnails and bullet should be from the right to left, also the left/right arrows functionality should be flipped, so the left button by default slides to the previous photo, but in RTL environment it slides to the next photo.